src/xia2/Driver/DriverFactory.py

```diff
@@ -12,21 +7,14 @@ def __init__(self):
 
         self._implemented_types = [
             "simple",
-            "script",
-            "interactive",
-            "qsub",
         ]
 
-        # should probably write a message or something explaining
-        # that the following Driver implementation is being used
-
-        if "XIA2CORE_DRIVERTYPE" in os.environ:
-            self.set_driver_type(os.environ["XIA2CORE_DRIVERTYPE"])
-
     def set_driver_type(self, driver_type):
         """Set the kind of driver this factory should produce."""
         if driver_type not in self._implemented_types:
             raise RuntimeError("unimplemented driver class: %s" % driver_type)
+        else:
+            raise RuntimeError("setting driver_type now unsupported")
 
         self._driver_type = driver_type
```

Comment:
It's probably better just to either add simply add a `DeprecationWarning` to the existing method, or replace the entire method with a single `RuntimeError`. The current approach results in a lgtm warning that the line `self._driver_type = driver_type` can never be hit.